platform.md

```diff
@@ -276,10 +303,34 @@ All lifecycle phases:
 - MUST read `CNB_PLATFORM_API` from the execution environment and evaluate compatibility before attempting to parse other inputs (see [Platform API Compatibility](#platform-api-compatibility))
 - MUST give command line inputs precedence over other inputs
 
+#### `preparer`
+Usage:
+```
+/cnb/lifecycle/preparer \
+  [-log-level <log-level>] \
+  [-stack-id <stack-id>] \
+  [-stack-path <stack-path>]
```

Comment:
Since this `preparer` will need registry credentials, I think we may need to be able to reference a builder. `--builder company/buildpacks` and pull the `stack.toml` from `io.buildpacks.builder.metadata`. This way you can run the scratch image for untrusted builders.